gstreamer/src/segment.rs

```diff
@@ -496,13 +496,16 @@ impl<T: FormattedValue> AsRef<Segment> for FormattedSegment<T> {
 impl<T: FormattedValue> fmt::Debug for FormattedSegment<T> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         let segment = self.as_ref();
+        let ptr = self as *const _;
         match segment.get_format() {
             Format::Undefined => f.debug_struct("Segment")
+                .field("ptr", &ptr)
```

Comment:
Printing the pointer here is not very useful, as the segment is stack allocated and moved around everywhere. So this value is going to change

Reply:
Removed.